src/apis/abusiveexperiencereport/v1.ts

```diff
@@ -257,6 +306,43 @@ export namespace abusiveexperiencereport_v1 {
     /**
      * abusiveexperiencereport.violatingSites.list
      * @desc Lists sites that are failing in the Abusive Experience Report.
+     * @example
+     * // Before running the sample:
+     * // - Enable the API at:
+     * //   https://console.developers.google.com/apis/api/abusiveexperiencereport.googleapis.com
+     * // - Login into gcloud by running:
+     * //   `$ gcloud auth application-default login`
+     * // - Install the npm module by running:
+     * //   `$ npm install googleapis`
+     *
+     * const {google} = require('googleapis');
+     * const abusiveexperiencereport = google.abusiveexperiencereport('v1');
+     *
+     * async function main() {
+     *   const auth = new google.auth.GoogleAuth({
+     *     // Scopes can be specified either as an array or as a single, space-delimited string.
+     *     scopes: [],
+     *   });
+     *
+     *   // Acquire an auth client, and bind it to all future calls
+     *   const authClient = await auth.getClient();
+     *   google.options('auth', authClient);
```

Comment:
I like that there are now inline examples, we should double check that our refdoc generator renders these appropriately at deploy time.

Reply:
Fun fact: inline samples are generally broken in compodoc 😭 
https://github.com/compodoc/compodoc/issues/304

The inline example is no more broken than it was before I guess?  I think @fhinkel is working on a plan to save us